controller/topics.controller.js

```diff
@@ -0,0 +1,13 @@
+const {
+    selectTopics
+} = require('../model/topics.model');
+
+const getTopics = (req, res, next) => {
+    const body = req.body
+    return selectTopics(body)
+    .then((result) => {
+        res.status(200).send(result)
```

Comment:
This should be an object with the key of topics and the results array as the value